crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -479,9 +478,9 @@ async fn submit_blocks_to_celestia(
         num_blocks = sequencer_block_data.len(),
         "submitting collected sequencer blocks to data availability layer",
     );
+
     let height = client
         .submit_sequencer_blocks(
-            SEQUENCER_NAMESPACE,
```

Comment:
This made me pause and think about the changes in the client. In principle this enables a relayer to relay blocks from multiple sequencers. This shouldn't happen because a sequencer should not willy-nilly change its chain ID, but I am not sure I am happy with making this a runtime feature.